src/hook/useAcademyInfoQuery.ts

```
@@ -0,0 +1,36 @@
+import { useQuery } from '@tanstack/react-query';
+import { useAcademyFilterStore } from '@store/academyFilterStore';
+import { useShallow } from 'zustand/react/shallow';
+import axios from 'axios';
+
+export const useAcademyInfoQuery = <TData = any>(pageNum: number) => {
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)